sim/dex-moves.ts

```diff
@@ -293,7 +293,8 @@ interface MoveHitData {
 }
 
 type MutableMove = BasicEffect & MoveData;
-export interface ActiveMove extends MutableMove {
+type RuinableMove = {[k in `ruined${'Atk' | 'Def' | 'SpA' | 'SpD'}`]?: boolean;};
+export interface ActiveMove extends MutableMove, RuinableMove {
```

Comment:
I feel like it would be better to just list these out individually like we do for the auras and -ate abilities, just to be consistent with said properties.

Reply:
Isn't that just because you can't have both *types* of auras affecting the same move?